bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2633,11 +2714,22 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
 
    }
 
-   else if ( name.substr(0,8) == "TVectorT" ) {  // allow proper iteration
+   else if (TString(name).Contains("TVectorT")) {
+      // allow proper iteration
       Utility::AddToClass( pyclass, "__len__", "GetNoElements" );
       Utility::AddToClass( pyclass, "_getitem__unchecked", "__getitem__" );
       Utility::AddToClass( pyclass, "__getitem__", (PyCFunction) CheckedGetItem, METH_O );
 
+      // add array interface
+      if (TString(name).Contains("<float>")) {
+         Utility::AddToClass(pyclass, "_get__array_interface__", (PyCFunction)TVectorArrayInterface<float, 4, 'f'>,
```

Comment:
Since these two calls to `AddToClass` and `AddProperty` are also repeated quite a bit, perhaps it would also be good to encapsulate them in an `InjectArrayInterface` function or similar.

Reply:
Done!